packages/usage-reporting-protobuf/package.json

```diff
@@ -5,19 +5,24 @@
   "type": "module",
   "exports": {
     ".": {
-      "import": "./generated/protobuf.mjs",
-      "require": "./generated/protobuf.cjs",
-      "types": "./generated/protobuf.d.ts"
+      "types": {
+        "require": "./generated/cjs/protobuf.d.ts",
+        "default": "./generated/esm/protobuf.d.ts"
+      },
+      "import": "./generated/esm/protobuf.js",
+      "require": "./generated/cjs/protobuf.js"
     }
   },
-  "main": "generated/protobuf.cjs",
-  "module": "generated/protobuf.mjs",
-  "types": "generated/protobuf.d.ts",
+  "main": "generated/cjs/protobuf.js",
+  "module": "generated/esm/protobuf.js",
+  "types": "generated/esm/protobuf.d.ts",
   "scripts": {
-    "generate": "rm -rf generated && mkdir generated && npm run pbjs-cjs && npm run pbjs-esm && npm run pbts",
-    "pbjs-cjs": "apollo-pbjs --target static-module --out generated/protobuf.cjs --wrap commonjs --force-number --no-from-object src/reports.proto",
-    "pbjs-esm": "apollo-pbjs --target static-module --out generated/protobuf.mjs --es6 --force-number --no-from-object src/reports.proto",
-    "pbts": "mv generated/protobuf.{c,}js && apollo-pbts -o generated/protobuf.d.ts generated/protobuf.js && mv generated/protobuf.{,c}js",
+    "generate": "rm -rf generated && mkdir generated && mkdir generated/esm && mkdir generated/cjs && npm run pbjs-cjs && npm run pbjs-esm && npm run pbts",
```

Comment:
```suggestion
    "generate": "rm -rf generated && mkdir -p generated/esm generated/cjs && npm run pbjs-cjs && npm run pbjs-esm && npm run pbts",
```

Reply:
Yeah I wasn't able to test that because that doesn't work on Windows
I don't have strong opinion, doesn't look like a lot of scripts are Windows friendly, I'm used to it by now